debian/control

```diff
@@ -60,7 +60,7 @@ Standards-Version: 3.9.3
 Package: ceph
 Architecture: linux-any
 Depends: binutils,
-         ceph-common (>= 0.78-500),
+         ceph-common (>= 0.94.2),
```

Comment:
It should be *&gt; 0.94.2* otherwise upgrading from 0.94.2 to 0.94.3 will not do the right thing.